content/en/blog/_posts/2021-03-30-Kubernetes-Multi-Tenancy-Usage-Models.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: blog
+title: 'Two Common Multi-Tenancy Models For Kubernetes'
+date: 2021-03-30
+slug: kubernetes-multi-tenancy-usage-models
+---
+
+**Authors:** Ryan Bezdicek (Medtronic), Jim Bugwadia (Nirmata), Tasha Drew (VMware),  Fei Guo (Alibaba), Adrian Ludwin (Google)
+
+Kubernetes clusters are typically used by several teams in an organization. Secure sharing of Kubernetes resources allows maximizing productivity and saving costs. In other cases, Kubernetes may be used to deliver software and platform services to end users requiring segmentation and isolation of resources across users.
+
+The Kubernetes Multi-Tenancy Working Group is chartered with defining multi-tenancy models for Kubernetes and making it easier to operationalize multi-tenant use cases.  This blog post, from the multi-tenancy working group members, describes two common usage models for multi-tenancy and introduces working group projects that can be used to simplify Kubernetes multi-tenancy management. 
+
+We will also be presenting on this content and discussing different use cases at our Kubecon EU 2021 panel session, [Multi-tenancy vs. Multi-cluster: When Should you Use What?](https://sched.co/iE66).
+
+## Multi-tenancy security considerations
+
+Cloud native security involves different system layers and lifecycle phases as described in the [Cloud Native Security Whitepaper](https://www.cncf.io/blog/2020/11/18/announcing-the-cloud-native-security-white-paper/) from CNCF SIG Security.  Without proper security measures implemented across all layers and phases, Kubernetes multi-tenancy can be compromised. 
+
+It is important for any new user to Kubernetes to realize that the default installation of a new upstream Kubernetes cluster is not secure, and you are going to need to invest in hardening it in order to avoid security issues.
+
+At a minimum, the following security measures are required:
+* Scan container images for vulnerabilities.
+* Apply [CIS Benchmarks for Kubernetes](https://www.cisecurity.org/benchmark/kubernetes/)
+* Restrict container image sources to trusted registries.
+* Implement granular [RBAC](/docs/reference/access-authn-authz/rbac/).
+* Apply default “deny-all” [network policies](/docs/concepts/services-networking/network-policies/).
+* Implement [Kubernetes Pod Security Standards](/docs/concepts/security/pod-security-standards/).
+
+Additional recommendations include:
+* Use a policy engine to validate configurations 
+* Implement runtime scanning
+* For stronger isolation, consider VM-based container sandboxing 
+
+While proper security is required independently of tenancy models, not having essential security controls like [pod security](/docs/concepts/security/pod-security-standards/) in a shared cluster provides attackers with means to compromise tenancy models and possibly access sensitive information across tenants increasing the overall risk profile.
+
+## Multi-tenancy usage models
```

Comment:
Can we consider moving this above the prior section? Explain what the models are, then talk about how to secure them. E.g. cluster-per-team might not need network policies or particularly granular RBAC.